workshops/git.md

```diff
@@ -280,7 +280,10 @@ If you do not have a GitHub account, you can create one **[here](https://github.
 
 ### **Create a GitHub personal access token**
 
-To push your local repo to GitHub, you will need a Personal Access Token. From your **GitHub** account, click through the following path to generate a new personal access token.  **Profile --> Settings --> Developer Settings --> Personal Access Tokens --> Tokens (classic) --> Generate new token (classic)**
+<details>
```

Comment:
Please change to not use inline HTML.

Also, please use mkdocs serve to review your desired changes.